### PR TITLE
Update workType labels for asian manuscripts

### PR DESCRIPTION
**Pull request by jamieparkinson**
Resolves https://github.com/wellcometrust/platform/issues/4218

### CODE REVIEW

File: common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/WorkType.scala [hunk 1]
```diff
@@ -155,7 +155,7 @@ object WorkType extends Enum[WorkType] {
 
   case object EManuscriptsAsian extends Linked {
     override val id: String = "x"
-    override val label: String = "E-manuscripts, Asian"
+    override val label: String = "Manuscripts"
```
Comment (jamesgorrie) [inline common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/WorkType.scala]: Is it worth leaving this here - and maybe leaving a note above saying what the Sierra label is?

**Comment by jamesgorrie** [comment]
Any reason not to have the `cinefilm` in here from wellcometrust/platform#4218?

File: common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/WorkType.scala (hunk 1 above)
Comment (jamieparkinson) [reply common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/WorkType.scala]: The Sierra label is captured in the name of the object imo

**Comment by jamieparkinson** [comment]
Re cinefilm: `n` is already called "Film". I guess that the request came from someone looking at Sierra rather than the API.

**Comment by jamesgorrie** [comment]
@jamieparkinson I was wondering if we keep what we're changing from in Sierra somewhere in this transformer.

**Pull request merged**